examples/alerting_example/public/components/create_alert.tsx

```diff
@@ -31,7 +31,7 @@ import { ALERTING_EXAMPLE_APP_ID } from '../../common/constants';
 export const CreateAlert = ({
   http,
   // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
Seems like it’s not necessary anymore to disable linting anymore for the next line, so this can be removed.

Reply:
@flash1293 . Thanks for the suggestion. I am not getting exactly what change I need to do. Could you help please

Reply:
You can remove the comment line (33) - it was just there to make the linter happy because underscores in a variable aren’t allowed.

Reply:
Thanks @flash1293. Made the changes